frontends/web/src/routes/account/receive/receive.jsx

```diff
@@ -129,30 +128,40 @@ export default class Receive extends Component {
         paired,
     }) {
         let uriPrefix = 'bitcoin:';
-        if (coinCode === 'ltc' || coinCode === 'tltc') {
+        if (code.startsWith('ltc') || code.startsWith('tltc')) {
             uriPrefix = 'litecoin:';
+        } else if (code.startsWith('eth') || code.startsWith('teth')) {
+            uriPrefix = '';
```

Comment:
I think 'ethereum' prefix would be fine/desirable: https://github.com/ethereum/EIPs/issues/67